integration_tests/mockApis/educationAndWorkPlanApi.ts

```diff
@@ -245,7 +245,7 @@ const stubGetTimeline = (prisonNumber = 'G6115VJ'): SuperAgentRequest =>
   stubFor({
     request: {
       method: 'GET',
-      urlPattern: `/timelines/${prisonNumber}`,
+      urlPathPattern: `/timelines/${prisonNumber}`,
```

Comment:
Needed to make this change to the wiremock stubs, because now the API is called with the query string params applied (all as false at the moment) - but it means as far as wiremock matching is concerned, it is a different URL so we need to use `urlPathPattern` rather than `urlPattern`